thrift/lib/cpp2/Makefile.am

```diff
@@ -173,7 +173,7 @@ libthriftprotocol_la_SOURCES = Version.cpp \
                                protocol/SimpleJSONProtocol.cpp \
                                protocol/VirtualProtocol.cpp
 
-libthriftprotocol_la_LIBADD = $(FOLLY_LIBS) $(BOOST_SYSTEM_LIB)
+libthriftprotocol_la_LIBADD = $(FOLLY_LIBS) $(BOOST_SYSTEM_LIB) -ldouble-conversion
```

Comment:
I wonder if the correct fix here is to make sure -ldouble-conversion gets included in $(FOLLY_LIBS), since folly depends on it.